frontend/src/components/forms/CreateLog.tsx

```diff
@@ -0,0 +1,394 @@
+/* eslint-disable prettier/prettier */
+import React, { useState } from "react";
+import Select, { MultiValue, SingleValue } from 'react-select'
+import {
+    Box,
+    Button,
+    Checkbox,
+    FormControl,
+    FormErrorMessage,
+    FormLabel,
+    Grid,
+    GridItem,
+    Input,
+    Modal,
+    ModalBody,
+    ModalContent,
+    ModalHeader,
+    ModalOverlay,
+    Text,
+    Textarea,
+} from "@chakra-ui/react";
+
+import { AddIcon } from '@chakra-ui/icons'
+import { SingleDatepicker } from "chakra-dayzed-datepicker";
+import { Col, Row } from "react-bootstrap";
+
```

Comment:
If you have too much time on your hands, putting these in state and setting them in a useEffect when the component renders would replicate an actual connection to the database more.

Reply:
I have no time 😭 good to note for the future though :D